apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +192,134 @@ export default class ExportHTMLModal extends Vue {
     showCode: false,
     exportType: '',
-    complianceCards: [],
+    // Series of icons used for profile-related detail reports
     icons: {
-      circleCheck: this.iconDatatoSVG(mdiCheckCircle, 'white'),
-      circleCross: this.iconDatatoSVG(mdiCloseCircle, 'white'),
-      circleMinus: this.iconDatatoSVG(mdiMinusCircle, 'white'),
-      circleAlert: this.iconDatatoSVG(mdiAlertCircle, 'white')
+      // Passed
+      circleCheck: this.iconHTMLStore.circleCheck,
+      // Failed
+      circleCross: this.iconHTMLStore.circleCross,
+      // Not applicable
+      circleMinus: this.iconHTMLStore.circleMinus,
+      // Not reviewed
+      circleAlert: this.iconHTMLStore.circleAlert,
+      // Profile error
+      triangleAlert: this.iconHTMLStore.triangleAlert,
+      // Total count
+      squareEqual: this.iconHTMLStore.squareEqual,
+      // No severity
+      circleNone: this.iconHTMLStore.circleNone,
+      // Low severity
+      circleLow: this.iconHTMLStore.circleLow,
+      // Medium severity
+      circleMedium: this.iconHTMLStore.circleMedium,
+      // High severity
+      circleHigh: this.iconHTMLStore.circleHigh,
+      // Critical severity
+      circleCritical: this.iconHTMLStore.circleCritical
     }
   };
 
+  // Configures outputData object's report type based on user input
   @Watch('exportType')
   onFileChanged(newValue: string, _oldValue: string) {
     switch (newValue) {
-      case 'executive':
-        this.description = 'Profile Info\nStatuses\nCompliance Donuts';
+      case FileExportTypes.executive:
+        this.description = FileExportDescriptions.executive;
         this.outputData.showControlSets = false;
         this.outputData.showCode = false;
         break;
-      case 'manager':
-        this.description =
-          'Profile Info\nStatuses\nCompliance Donuts\nTest Results and Details';
+      case FileExportTypes.manager:
+        this.description = FileExportDescriptions.manager;
         this.outputData.showControlSets = true;
         this.outputData.showCode = false;
         break;
-      case 'administrator':
-        this.description =
-          'Profile Info\nStatuses\nCompliance Donuts\nTest Results and Details\nTest Code';
+      case FileExportTypes.administrator:
+        this.description = FileExportDescriptions.administrator;
         this.outputData.showControlSets = true;
         this.outputData.showCode = true;
         break;
     }
   }
 
-  iconDatatoSVG(
+  // Generate SVG HTML for icons for injection into export template
+  iconDataToSVG(
     iconData: string,
     fill: string,
     widthPx = 24,
     heightPx = 24
   ): string {
-    return `<svg style="width:${widthPx}px; height:${heightPx}px" viewBox="0 0 ${widthPx} ${heightPx}"><path fill="${fill}" d="${iconData}"/></svg>`;
+    return `<svg style="width:${widthPx}px; height:${heightPx}px" viewBox="0 0 ${widthPx} ${heightPx}" role="img" aria-hidden="true"><path fill="${fill}" d="${iconData}"/></svg>`;
```

Comment:
instead of always setting it to be hidden here, i would make it respond to a parameter / passed in alt-text with the default behavior being to set it to hidden.  and then we can leave a comment where we define all of our icons to say that since they're purely supplementary we don't need to provide alt text.

Reply:
Changed as requested.